src/components/elements/dropdown.js

```diff
@@ -40,6 +40,9 @@ const Symbol = styled(Flex)`
 `
 
 export const DropdownContainer = styled.ul`
+    @media (width: 428px) {
+        height: 46px;
+    }
```

Comment:
can we use one of the sizes from `device.ts` ?

Reply:
Yes